### DISCUSSION TITLE
How to display with various runtimes ?

### DISCUSSION THREAD

**Discussion by waghanza** [SuspiciousLookingOwl/web-frameworks-benchmark#24] [Show and tell]
Hi @SuspiciousLookingOwl,

I plan to test framework with runtimes in https://github.com/the-benchmarker/web-frameworks/pull/3958.

For example, use `laravel` through classic `php-fpm` or some new runtimes like `swoole`.

Any idea on how to display this ?

Regards,

**Comment by SuspiciousLookingOwl** [comment]
How about something like this:

| Language | Runtime | Framework | RPS (64) | RPS(256) | RPS(512) |
| --- | --- | --- | --- | --- | --- |
| PHP | php-fpm | Laravel | 1 | 1 | 1 |
| PHP | swoole | Laravel | 2 | 2 | 2 |

**Reply by waghanza** [reply]
Don't you think this will decrease readability.

How about :
+ display by default only what is recommended, for example display only results with `puma` or `rails`
+ having a list of runtimes available per language + a way to display result with them 

Perhaps the second bullet could be a toggle, either default runtime or all runtimes
